samples/pan-bar.html

```diff
@@ -3,9 +3,9 @@
 
 <head>
 	<title>Bar Chart Pan</title>
-	<script src="../node_modules/chart.js/dist/Chart.bundle.js"></script>
-	<script src="../node_modules/hammerjs/hammer.min.js"></script>
-	<script src="../dist/chartjs-plugin-zoom.js"></script>
+	<script src="https://cdn.jsdelivr.net/npm/chart.js@2.7.3/dist/Chart.min.js"></script>
+	<script src="https://cdn.jsdelivr.net/npm/hammerjs@2.0.8/hammer.min.js"></script>
```

Comment:
That are the links I sent you on Slack but actually we don't need the extra file path for Hammer. It will be the same with Chart.js starting v2.8, so this would become:

```html
<script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
<script src="https://cdn.jsdelivr.net/npm/hammerjs@2.0.8"></script>
<script src="../dist/chartjs-plugin-zoom.min.js"></script>
```